common/desktop-exports

```diff
@@ -289,6 +289,10 @@ function make_user_fontconfig {
   if [ -d $REALHOME/.cache/fontconfig ]; then
     echo "  <cachedir>$REALHOME/.cache/fontconfig</cachedir>"
   fi
+  
+  # fix font render for modified fonts, that discussed on Snapcraft Forum:
+  # https://forum.snapcraft.io/t/snap-package-cannot-read-fonts-conf/16657
+  echo '  <include ignore_missing="yes">/etc/fonts/conf.d</include>'
```

Comment:
This line should go before `echo '  <include ignore_missing="yes">conf.d</include>'` to allow user to override `global` config by `user` specific one

Reply:
Thank you, I resolve it :smile: